automated-comments/go/hamming/declare_variable_when_needed_not_start_of_function.md

```diff
@@ -0,0 +1 @@
+You are declaring `%{returnVar}` before you check if the length is equal, so the declaration may end up being unnecessary. It's better to declare a variable right before you use it.
```

Comment:
As mention below: `%{returnVar}` is a placeholder for the actual variable name used by the student.

Reply:
Suggestion with variable name:

> Variably `%returngVar}` is declared early. It's common when writing Go to declare variables as late as possible before they are used. This tends to reduce mental overhead when reading the code, and sometimes also leads to simpler code, since the code might return before the variable is needed.

Note: Don't like this one too much though. Maybe someone has a better suggestion how it integrate the variable name here.